examples/src/main/java/com/datastax/oss/driver/examples/json/jsr/Jsr353JsonCodec.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.datastax.oss.driver.examples.json.codecs;
+package com.datastax.oss.driver.examples.json.jsr;
```

Comment:
Moved this codec here since he was alone in its old package and there was no reason why we wouldn't include it here along with the examples that use it.